TramsDataApi/Migrations/TramsDb/TramsDbContextModelSnapshot.cs

```diff
@@ -583,6 +583,21 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                     b.ToTable("A2BSchoolLoan","sdd");
                 });
 
+            modelBuilder.Entity("TramsDataApi.DatabaseModels.A2BSelectOption", b =>
```

Comment:
I know this file is probably auto generated but just flagging this to check if it is intentional as looks like A2B